extensions/2.0/Vendor/MESHOPT_compression/README.md

```diff
@@ -0,0 +1,132 @@
+# MESHOPT\_compression

```

Comment:
Consider `MESHOPT_mesh_compression`. 😇

Reply:
Note that this extension is more general and allows compressing animation data as well. Maybe `view_compression` would be appropriate?

Reply:
Ah, good point. In that case `view_compression` sounds reasonable, or I would be open to considering this an exception to the `<scope>_<feature>` guideline too.